src/api/app/policies/flag_policy.rb

```diff
@@ -0,0 +1,7 @@
+class FlagPolicy < ApplicationPolicy
+  # just admin are able to create flag
+  def create?
+    return true if user.is_admin?
+    return true unless ['sourceaccess', 'access'].include?(record.flag)
```

Comment:
```
return true unless X
```

at the end of a (ruby) function is basically long for 

```
!X
```

Reply:
The current code could return `nil`, but I don't see why it shouldn't be a boolean instead.

It could also be rewritten this way:

```
user.is_admin? || ['sourceaccess', 'access'].exclude?(record.flag)
```